MapboxNavigation/CarPlayNavigationViewController.swift

```diff
@@ -316,6 +316,13 @@ public class CarPlayNavigationViewController: UIViewController {
             return CGRect(x: 0, y: 0, width: widthOfManeuverView, height: 30)
         }
         
+        // Over a certain height, CarPlay devices downsize the image and CarPlay simulators hide the image.
+        let maximumImageSize = CGSize(width: .infinity, height: shieldHeight)
```

Comment:
IIRC, the image max height of an attachment in  an attributed string drawn by a UILabel should be `fontSize.height + font.ascender + font.descender`.

Reply:
As far as I can tell, none of the accessibility settings on iOS affect the font size of the instructions in CarPlay. CarPlay units do have their own settings (such as the left/right layout setting), but I haven’t seen a setting about the font size in the units I’ve used so far.

Reply:
Hard limit before image gets cropped or scaled depending on other attributes. I thought we had some leeway but, if there seem to be a 16p limit, let's stick with that.